### DISCUSSION TITLE
Accessible documentation

### DISCUSSION THREAD

**Discussion by xaocon** [synodriver/aioaria2#24] [Ideas]
Hi,

First off, great project! Thanks for the work you do on it.

Since this is such a useful project I was wondering if I could help in some way to make the documentation more accessible. I really appreciate you already including all the english you have. Maybe using readthedocs, sphinx, and transifex could help? I haven't used them before but would be happy to try to help in some way. I think just the examples and readmes you already have and then grabbing the doc strings and making them available in english would be a great start but others could help add other languages. If that's too much work maybe you'd accept a PR to change to docstrings to follow pep 8 or one that has some english info as well?

**Comment by synodriver** [comment]
Sorry for the late reply. Prs are always welcome! As for pep8, it follows the original rpc method name, I think we'd better keep it？
